Populate dedication page, add dates to prevent page rot, and add last updated date (#73)
CurtisGreen/dallas-area-transit-alliance: _posts/remembering-mary.md

banner: 
  text: "Remembering Mary - A Friend, Advocate, and Champion of DATA. Your passion and commitment will always be remembered."
---

> Yesterday (January 24, 2025), we found out that Mary Collier, our Events Planning Committee Chair passed away on Tuesday (January 21, 2025).

Reviewer: ```suggestion
Yesterday (January 24, 2025), we found out that Mary Collier, our Events Planning Committee Chair, passed away on Tuesday (January 21, 2025).
```

Mary was a fierce warrior for public transportation while also planning so many social gatherings that brought this community together.

> Mary has several planned events in the [calendar](/calendar), and we realize that may raise some important questions. The Executive Committee has decided that out of respect for her and all the work she put into DATA, those events will not be cancelled. It is our belief that she would have wanted us to keep the community together and continue our efforts to save DART, so that is exactly what we will be doing.

Reviewer 2: thank you

> Mary has several planned events in the [calendar](/calendar), and we realize that may raise some important questions. The Executive Committee has decided that out of respect for her and all the work she put into DATA, those events will not be cancelled. It is our belief that she would have wanted us to keep the community together and continue our efforts to save DART, so that is exactly what we will be doing.

Reviewer: ```suggestion
Mary had planned several events in the [calendar](/calendar), and we realize that may raise some important questions. The Executive Committee has decided that out of respect for her and all the work she put into DATA, those events will not be cancelled. It's our belief that she would have wanted us to keep the community together and continue our efforts to save DART, so that's exactly what we will be doing.
```

We will miss her dearly. 

===

> This post will be updated, when we have more information to share and with Mary's legacy for DATA.

Reviewer 2: bad comma
